style.css

```diff
@@ -84,3 +85,615 @@
   left: 0;
   top: 472px;
 }
+
+.workSection {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
As requested in the project requirements:
![image](https://user-images.githubusercontent.com/42799579/198420726-76f7621c-3743-4f2c-adc7-e3338045a9d9.png)
Please make sure to use both a grid and flexboxes to organize your projects section 😃.

Reply:
![Screenshot 2022-10-28 at 13 40 43](https://user-images.githubusercontent.com/37203742/198590247-99f973f7-0c82-42bc-b55b-e39a47972219.png)